assets/src/components/Treeview.js

```diff
@@ -27,9 +26,42 @@ export default class Treeview extends HTMLElement {
     connectedCallback() {
 
         this._onChange = () => {
-            render(this._layerTemplate(mainLizmap.state.layerTree), this);
+            render(this._rootTemplate(mainLizmap.state.layerTree), this);
         };
 
+        this._olLayerTemplate = (olLayer) =>
+            html`
+        <li data-testid="ol-${olLayer.name}">
+            <div class="${olLayer.checked ? 'checked' : ''} ${olLayer.type}">
+                <div class="loading ${olLayer.loadStatus === MapLayerLoadStatus.Loading ? 'spinner' : ''}"></div>
+                <input type="checkbox" id="node-ol-${olLayer.name}" .checked=${olLayer.checked} @click=${() => olLayer.checked = !olLayer.checked} >
+                <div class="node ${olLayer.isFiltered ? 'filtered' : ''}">
+                    <img class="legend" src="${olLayer.icon}">
+                    <label for="node-ol-${olLayer.name}">${olLayer.wmsTitle}</label>
+                    <div class="layer-actions">
+                    </div>
+                </div>
+            </div>
+        </li>`
+
+        this._extGroupTemplate = (extGroup) =>
+            html`
+        <li data-testid="ext-${extGroup.name}">
+            <div class="expandable ${extGroup.expanded ? 'expanded' : ''}" @click=${() => extGroup.expanded = !extGroup.expanded}></div>
+            <div class="${extGroup.checked ? 'checked' : ''} ${extGroup.type} group">
```

Comment:
```suggestion
            <div class="${extGroup.checked ? 'checked' : ''}  group">
```

Reply:
It seems `${extGroup.type}` = ext-group. If yes you can ignore my suggestion.

Reply:
extGroup.type is 'ext-group'